### ISSUE TITLE
Chrome Password manager bug

### ISSUE THREAD

**Issue by saintjimmygd** [microg/GmsCore#2393]
I see a problem in latest microG. When using Chrome v.123,124,125, chrome's Password manager won't work/open. Sites do not offer to save passwords anymore and autofill passwords don't work. It works on older Chrome builds like v.122 and below. However, using older microG 0.3.0, latest Chrome works as it should, everything is fine. Please fix. Android 14 qpr2.

**Comment by dimka40185** [comment]
huawei has the same problem, the password manager in the new chrome builds does not work

**Comment by saintjimmygd** [comment]
On vanilla rom and latest microG. Installed latest Chrome from Aurora, password manager still not working (tapping on it doesn't do anything). Uninstalled chrome, installed older v.122 Chrome from apkmirror, password manager works. Was prompted in Chrome for the v.125 update, so I did it through Aurora. Chrome updated to latest, password manager WORKS NOW . Don't know what's going on anymore with this, but happy it's working ! 🥳

**Comment by Wolfeg** [comment]
> On vanilla rom and latest microG. Installed latest Chrome from Aurora, password manager still not working (tapping on it doesn't do anything). Uninstalled chrome, installed older v.122 Chrome from apkmirror, password manager works. Was prompted in Chrome for the v.125 update, so I did it through Aurora. Chrome updated to latest, password manager WORKS NOW . Don't know what's going on anymore with this, but happy it's working ! 🥳

thx, that workaround helped

**Comment by luciending** [comment]
However the update through Aurora lags significantly

**Comment by leisurefire** [comment]
> On vanilla rom and latest microG. Installed latest Chrome from Aurora, password manager still not working (tapping on it doesn't do anything). Uninstalled chrome, installed older v.122 Chrome from apkmirror, password manager works. Was prompted in Chrome for the v.125 update, so I did it through Aurora. Chrome updated to latest, password manager WORKS NOW . Don't know what's going on anymore with this, but happy it's working ! 🥳

This is because MicroG hasn't implemented Google Password Manager (Soon I hope?), yet Google is trying to move the whole Chrome password manager to GMS. On previous builds, these entries or components are not fully removed.

**Comment by TNT-Likely** [comment]
> On vanilla rom and latest microG. Installed latest Chrome from Aurora, password manager still not working (tapping on it doesn't do anything). Uninstalled chrome, installed older v.122 Chrome from apkmirror, password manager works. Was prompted in Chrome for the v.125 update, so I did it through Aurora. Chrome updated to latest, password manager WORKS NOW . Don't know what's going on anymore with this, but happy it's working ! 🥳

thanks ,it work fine for me

**Comment by xlionjuan** [comment]
I'm having same problem but with Bitwarden password manager, the Chrome+Gboard never showing Bitwarden prompts either on Chrome or Gboard, but works on Firefox Android.

**Comment by ale5000-git** [comment]
Maybe this PR help: #2652

**Comment by zZavarivach** [comment]
The workaround with chrome v1.22 works, cheers

**Comment by artquest** [comment]
Unfortunately v.122 workaround doesn't work for me (Huawei Nova 12s). I uninstalled the latest version and installed v.122 from APKMirror — passwords sync worked. But when I updated Chrome to the latest version (via Aurora) the sync stopped working.

**Comment by ale5000-git** [comment]
> Unfortunately v.122 workaround doesn't work for me (Huawei Nova 12s). I uninstalled the latest version and installed v.122 from APKMirror — passwords sync worked. But when I updated Chrome to the latest version (via Aurora) the sync stopped working.

Have you tried with microG v0.3.9?

**Comment by artquest** [comment]
> Have you tried with microG v0.3.9?

Thank you, I've updated both Services and Companion to the latest prerelease (v0.3.9.250932), but unfortunately it didn't help — still no passwords sync in Chrome.

**Comment by residentcode** [comment]
microG stilll not fix Chrome Password manager bug more than a year already.

**Comment by Idcrafter** [comment]
there is a better app to test it now and it's the "standalone" password manager app that is probably just linking to gms to open the password manager UI like what chrome probably did.

**Comment by residentcode** [comment]
> password manager

Google Password Manager doesn't even start on huawei

**Comment by Idcrafter** [comment]
yeah because it is just a link that opens it from inside the normal play services. microG seems to not have anything implemented for it so it can't be opened.

**Comment by teamfighter** [comment]
Any updates here?

**Comment by ovkhasch** [comment]
I confirm that I can use the password manager in chrome 122 but it stops working after updating chrome, no matter which app store is used to update. It's hard to believe that autofill is broken for so long for so many people!

**Comment by vaimeo** [comment]
Same problem here EMUI 15, Mate 80 Pro, chrome version 148
